.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "gir"]
 	path = gir
-	url = https://github.com/gtk-rs/gir
+	url = https://github.com/sdroege/gir
```

Comment:
Don't forget to revert this one.

Reply:
Yeah that's why those commits are marked WIP :)